packages/utils/src/Crypto.ts

```diff
@@ -42,14 +41,28 @@ import jsonabc from './jsonabc.js'
 import * as SDKErrors from './SDKErrors.js'
 import { ss58Format } from './ss58Format.js'
 
-export {
-  naclBoxPairFromSecret,
-  mnemonicGenerate,
-  mnemonicToMiniSecret,
-} from '@polkadot/util-crypto'
+export { mnemonicGenerate, mnemonicToMiniSecret } from '@polkadot/util-crypto'
 
 export { encodeAddress, decodeAddress, u8aToHex, u8aConcat }
 
+/**
+ * Creates a new public/secret box keypair from a secret.
+ *
+ * @param secret The secret.
+ * @returns An object containing a box `publicKey` & `secretKey` generated from the supplied secret.
+ * @example
+ * <BR>
+ *
+ * ```javascript
+ * import { naclBoxPairFromSecret } from '@polkadot/util-crypto';
```

Comment:
We export the function from Crypto, but its example imports it from polkadot, this is a bit confusing

Reply:
ohhhh yes got spot

Reply:
we don't usually do examples, I'll just remove it